docs/docs/getting-started/first-app.md

```diff
@@ -4,9 +4,9 @@ sidebar_position: 4
 
 # Write your first app
 
-Let's create a simple converational Agent using Jaseci and Jaseci Kit. We're gonna creat a Chat bot for students to sign up for Jaseci Dojo !
+Let's create a simple conversational Agent using Jaseci and Jaseci Kit. We're gonna create a Chatbot for students to sign up for Jaseci Dojo !
 
-Before we begin ensure your have jaseci and jaseci kit installed. If not see the Installation here.
+Before we begin ensure you have jaseci and jaseci kit installed. If not, see the Installation here.
```

Comment:
Before we begin ensure you have Jaseci and Jaseci Kit installed. If not, see the Installation here.